libraries/chain/include/eosio/chain/exceptions.hpp

```diff
@@ -257,6 +257,10 @@ namespace eosio { namespace chain {
                                     3030013, "Block includes an ill-formed additional block signature extension" )
       FC_DECLARE_DERIVED_EXCEPTION( invalid_qc_claim, block_validate_exception,
                                     3030014, "Block includes an invalid QC claim" )
+      FC_DECLARE_DERIVED_EXCEPTION( invalid_qc, block_validate_exception,
+                                    3030014, "Block includes an invalid QC" )
+      FC_DECLARE_DERIVED_EXCEPTION( invalid_qc_signature, block_validate_exception,
+                                    3030014, "Block includes a QC with invalid signature(s)" )
```

Comment:
Change the exception #. Also the # for `invalid_qc`

Reply:
https://github.com/AntelopeIO/spring/pull/927/commits/d134711fe9ba076f5a14084e64c3be2e1e5318fd